src/components/pages/MapPage/index.js

```diff
@@ -65,29 +149,47 @@ function MapPage() {
     if (Object.keys(boundary).length) {
       getFeedLocation();
     }
-  }, [zoomLevel, boundary]);
+  }, [zoomLevel, boundary, ploggingResult]);
 
   return (
-    <MapTemplate>
-      {defaultProps && (
-        <GoogleMapReact
-          defaultCenter={defaultProps.center}
-          defaultZoom={defaultProps.zoom}
-          // bootstrapURLKeys={{ key: process.env.REACT_APP_MAP_API }}
-          onChange={({ zoom, bounds }) => {
-            setZoomLevel(zoom);
-            setBoundary({
-              NWlatitude: bounds.nw.lat,
-              NWlongitude: bounds.nw.lng,
-              SElatitude: bounds.se.lat,
-              SElongitude: bounds.se.lng,
-            });
-          }}
-        >
-          {spreadFeeds()}
-        </GoogleMapReact>
+    <>
+      <MapTemplate>
+        {defaultProps && (
+          <GoogleMapReact
+            defaultCenter={defaultProps.center}
+            defaultZoom={defaultProps.zoom}
+            // bootstrapURLKeys={{ key: process.env.REACT_APP_MAP_API }}
+            onChange={({ zoom, bounds, ...props }) => {
+              setZoomLevel(zoom);
+              setBoundary({
+                NWlatitude: bounds.nw.lat,
+                NWlongitude: bounds.nw.lng,
+                SElatitude: bounds.se.lat,
+                SElongitude: bounds.se.lng,
+              });
+            }}
+          >
+            {spreadFeeds()}
+          </GoogleMapReact>
+        )}
+      </MapTemplate>
+      {modalClick && modalInfo && (
+        <Portal wrapperId="modal-container">
+          <QuestTemplate
+            onCloseClick={() => closeModalButton()}
```

Comment:
위에 선언해둔 closeModalButton은 setModalClick(false); 만 실행시키니까 이곳에서 setModalClick(false);을 실행시켜도 괜찮을 것 같습니다~!
```suggestion
            onCloseClick={() => setModalClick(false)}
```

Reply:
이렇게 할 수도 있군요~~ 배워갑니다